src/client/components/military-service/ServiceInformationSection.jsx

```diff
@@ -12,15 +12,33 @@ import { veteranUpdateField } from '../../actions';
  * `reviewSection` - Boolean. Hides components that are only needed for ReviewAndSubmitSection.
  */
 class ServiceInformationSection extends React.Component {
+  constructor() {
+    super();
+    this.displayLabel = this.displayLabel.bind(this);
+  }
+
+  displayLabel(fullList, userValue) {
```

Comment:
Ah, this is exactly the same as the other function. Perhaps we can export it to a shared utils file, perhaps even in `calculated.js` and then just import it in the files you need?